src/app/(pages)/(SigleColumn)/register/page.jsx

```diff
@@ -0,0 +1,117 @@
+"use client";
+
+import Breadcrumb from "@/components/common/Breadcrumb";
+import PageTitle from "@/components/common/PageTitle";
+import { getLabels } from "@/components/common/fetchData";
+import Link from "next/link";
+import { useRef, useState } from "react";
+import { register } from "@/components/common/fetchData";
+import { useUser } from "@/components/common/userContext";
+import { useRouter } from "next/navigation";
+import AlertError from "@/components/ui/AlertError";
+
+export default function Register() {
+  const contentDirectory = getLabels();
+  const content = contentDirectory.register;
+  const { user, loading, storeUser } = useUser();
+  const router = useRouter();
+  const [alert, setAlert] = useState(false);
+
+  const name1 = useRef("");
+  const name2 = useRef("");
+  const email = useRef("");
+  const login_pwd = useRef("");
+
+  const handleChange = () => {
+    setAlert(false);
+  };
+
+  const handleRegister = async (event) => {
+    event.preventDefault();
+
+    const user = await register(
+      name1.current.value,
+      name2.current.value,
+      email.current.value,
+      login_pwd.current.value
+    );
+    console.log(user);
+
+    if (user) {
+      setAlert(false);
+      storeUser(user);
+      router.push("member/mypage");
+    } else {
+      setAlert(true);
+    }
+  };
+
+  return (
+    <div className="l-container">
+      <Breadcrumb paths={[{ label: content.text }]} />
+      <PageTitle content={content} />
+      <div className="l-container--small l-container--contents">
+        <div className="c-form-group u-text-align-center">
+          <p className="c-text--small">
+            <span className="c-form-label__required">*</span>は必須項目です。
+          </p>
+        </div>
+        <form className="c-form" onSubmit={handleRegister} onChange={handleChange}>
+          <div className="c-form-group">
+            <label htmlFor="name1" className="c-form-label">
+              名前（姓）
+            </label>{" "}
```

Comment:
please set style to avoid `{" "}`

Reply:
Done https://github.com/diverta/front_next_media/pull/4/commits/fe59861784d4a94c4e49083f3d68e224193d301d